wagtail/tests/settings.py

```diff
@@ -22,6 +22,9 @@
     }
 }
 
+if DATABASES['default']['ENGINE'] == 'sql_server.pyodbc':
```

Comment:
@saevarom This should be `django.db.backends.sqlite3` I think...?

Reply:
Woops, your right. Paste error :)

Reply:
I'll try again